### PR TITLE
Feat/community account name

### PR DESCRIPTION
**Pull request by danielAlvess**
This merge did: 

- implement community name is sub account of premium account `c`
- fix check right holder OR logic

### CODE REVIEW

File: community/src/community.cpp [hunk 1]
```diff
@@ -10,6 +10,7 @@ const uint64_t init_ram_amount = 10 * 1024;
 const asset stake_net = asset(1'0000, CORE_SYMBOL);
 const asset stake_cpu = asset(1'0000, CORE_SYMBOL);
 
+const name community_name_creator = "c"_n;
```
Comment (quocle108) [inline community/src/community.cpp]: "c............"_n

File: community/src/community.cpp [hunk 2]
```diff
@@ -2001,45 +1988,49 @@ bool community::is_pos_candidate(name community_account, uint64_t pos_id, name o
...
         }
     }
 
-    return is_right_badge;
+    return false;
+}
+
+bool community::verify_community_account_input(string community_account) {
+    uint64_t account_length = community_account.length();
+    if (account_length < 6 || 12 < account_length) {
```
Comment (quocle108) [inline community/src/community.cpp]: account_length > 13

**Comment by danielAlvess** [comment]
@quocle108 pls review again!.

File: community/src/community.cpp [hunk 3]
```diff
@@ -5,11 +5,12 @@
 
 const symbol ramcore_symbol = symbol(symbol_code("RAMCORE"), 4);
 const symbol ram_symbol = symbol(symbol_code("RAM"), 0);
-const symbol CORE_SYMBOL = symbol(symbol_code("CAT"), 4);
+const symbol CORE_SYMBOL = symbol(symbol_code("SYS"), 4);
```
Comment (quocle108) [inline community/src/community.cpp]: CAT

File: community/src/community.cpp [hunk 4]
```diff
@@ -2001,45 +1988,44 @@ bool community::is_pos_candidate(name community_account, uint64_t pos_id, name o
...
 
-    return is_right_badge;
+    return false;
+}
+
+bool community::verify_community_account_input(name community_account) {
+    if (community_account.suffix() != community_name_creator) {
+        return false;
+    }
+
```
Comment (quocle108) [inline community/src/community.cpp]: community_account >=6 characters

File: community/src/community.cpp [hunk 5]
```diff
@@ -2001,45 +1988,46 @@ bool community::is_pos_candidate(name community_account, uint64_t pos_id, name o
...
+            return true;
         }
     }
 
-    return is_right_badge;
+    return false;
+}
+
+bool community::verify_community_account_input(name community_account) {
+    if (community_account.length() < 6) return false;
```
Comment (quocle108) [inline community/src/community.cpp]: i think community_account.length() alway >6

**Pull request merged**